src/Momento.Sdk/Momento.Sdk.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<!-- Build Configuration -->
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
it's okay if we don't know the answer to this until we play with the released version a bit, but i'm curious which of these will be chosen as the framework when building for .NET Framework 4.8.x

Reply:
It uses `net461`. I had tested this already with the integration tests. That is, I can set the integration tests to run on .NET Framework 4.8. It will select the binary built on `net461`. See https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting

Reply:
lovely

Reply:
i think it would be worth adding a comment/link on the `#if NETFRAMEWORK` lines to make it super clear when that condition is satisfied, but not a blocker.

Reply:
I refactored this part in this commit. That conditional is no longer in the code and is only in the csproj file now. See https://github.com/momentohq/client-sdk-dotnet/pull/426/commits/e6276718d20e34ed5899f56a0dbef8541ce4a872